package.json

```diff
@@ -74,15 +81,12 @@
     "html-webpack-plugin": "^2.30.1",
     "jest": "^21.2.1",
     "jest-fetch-mock": "^1.2.1",
-    "less": "^2.7.1",
-    "less-loader": "^2.2.3",
     "prettier": "^1.4.4",
     "react-hot-loader": "^1.3.0",
     "react-scripts": "1.0.7",
-    "react-test-renderer": "^15.6.1",
+    "react-test-renderer": "^16.3.2",
     "redux-mock-store": "^1.2.3",
     "relay-compiler-webpack-plugin": "^0.3.0",
-    "style-loader": "^0.13.1",
     "url-loader": "^0.5.7",
     "webpack": "^1.13.1",
```

Comment:
do we need to upgrade webpack?
``````npm WARN codemirror-graphql@0.6.11 requires a peer of graphql@^0.6.0 || ^0.7.0 || ^0.8.0-b || ^0.8.0 || ^0.9.0 || ^0.10.0 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-react-app@1.0.5 requires a peer of babel-eslint@^7.2.3 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-react-app@1.0.5 requires a peer of eslint@^3.19.0 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-loader@1.7.1 requires a peer of eslint@>=1.6.0 <4.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN extract-text-webpack-plugin@2.1.0 requires a peer of webpack@^2.2.0 but none is installed. You must install peer dependencies yourself.
npm WARN material-ui-icons@1.0.0-beta.17 requires a peer of material-ui@^1.0.0-beta.16 but none is installed. You must install peer dependencies yourself.
npm WARN redux-devtools@3.4.0 requires a peer of react@^0.14.9 || ^15.3.0 but none is installed. You must install peer dependencies yourself.
npm WARN redux-form@6.8.0 requires a peer of react@^15.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN relay-compiler-webpack-plugin@0.3.0 requires a peer of webpack@>=2.3.0 but none is installed. You must install peer dependencies yourself.
npm WARN schema-utils@0.4.5 requires a peer of webpack@^2.0.0 || ^3.0.0 || ^4.0.0 but none is installed. You must install peer dependencies yourself.```